base/cvd/cuttlefish/host/libs/vm_manager/crosvm_display_controller.cpp

```diff
@@ -107,7 +107,9 @@ Result<int> CrosvmDisplayController::RunCrosvmDisplayCommand(
 
   Result<std::string> res = RunAndCaptureStdout(std::move(command));
   if (res.ok()) {
-    *stdout_str = CF_EXPECT(std::move(res));
+    if (stdout_str != nullptr) {
+      *stdout_str = CF_EXPECT(std::move(res));
```

Comment:
nit: since this is already in a `res.ok()` block, it could also be `*stdout_str = std::move(*res);`

Reply:
Done.